pyproject.toml

```diff
@@ -19,12 +19,13 @@ numpy = "^1.23.3"
 scipy = "^1.9.1"
 Cython = "^0.29.32"
 scikit-fmm = "^2022.8.15"
-scikit-image = "^0.18"
+scikit-image = ">0.18"
```

Comment:
Can we pin it to the latest version that you verified to work? Just in case a later change in scikit-image silently breaks it.

Reply:
otherwise the PR looks good.

Reply:
It's working for me with skimage v0.19.3 -- should a max version be added and change the dependcy to ">0.18, <0.20"?

Reply:
we can just pin it to 0.19.3 for sanity.

Reply:
that works -- would like me to make the change and commit it?

Reply:
yes. pls go ahead. i ll approve after the commit

Reply:
done!